src/stacks-hierarchy/types.ts

```diff
@@ -1,36 +1,23 @@
 import {PaymentType} from '@atb/ticketing';
 
-export type CardPaymentMethod =
-  | {paymentType: PaymentType.Visa | PaymentType.Mastercard; save: boolean}
-  | {
-      paymentType: PaymentType.Visa | PaymentType.Mastercard;
-      recurringPaymentId: number;
-    };
-
-export type VippsPaymentMethod = {
-  paymentType: PaymentType.Vipps;
-};
-
-export type PaymentMethod = VippsPaymentMethod | CardPaymentMethod;
-
-export type SavedRecurringPayment = {
+export type RecurringCard = {
   id: number;
   expires_at: string;
   masked_pan: string;
   payment_type: number;
 };
 
-export type DefaultPaymentOption = {
-  savedType: 'normal';
-  paymentType: PaymentType;
+type CardPaymentMethod = {
+  savedType: 'recurring' | 'normal';
```

Comment:
Could have a `savedType` as an enum, with explanations as well ("normal" isn't the most self-explanatory type ever 😛)

Reply:
Sure, does something like this work? 

```ts
export enum SavedPaymentMethodType {
  /**
   * Contains only the type of payment, and no payment details. When used, the
   * user has to enter the payment details manually or pay through a third party
   * like Vipps.
   */
  Normal = 'normal',
  /**
   * Contains the type of payment and information to enable automatic payment
   * without entering more payment details. BankID might still be required.
   */
  Recurring = 'recurring',
}
```

Reply:
Great!